lib/adapters/maintainer_adapter.spec.ts

```diff
@@ -56,8 +56,8 @@ describe("maintainerAdapter", () => {
   })
 
   afterEach(() => {
-    role = null
-    user = null
+    role = null!
+    user = null!
```

Comment:
null assertions on assignments don't really make sense

Reply:
Yeah but it was needed for null is not assignable error , solution is to assign non-null assertion operator.

Reply:
just saw the rest of the file. just use `O.none`